crc/api.yml

```diff
@@ -441,13 +441,36 @@ paths:
       responses:
         '200':
           description: list of Study Associate Objects
```

Comment:
It looks like you lost this part of the responses code here for study associates

          content:
            application/json:
              schema:
                type: array
                items:
                  $ref: "#/components/schemas/StudyAssociate"